src/enkf/gridinfo_fv3reg.f90

```diff
@@ -43,7 +43,8 @@ module gridinfo
 !
 !$$$
 
-use mpisetup, only: nproc, mpi_integer, mpi_real4, mpi_comm_world,mpi_status
+use mpisetup, only: nproc, mpi_integer, mpi_real4,mpi_status
+use mpimod, only: mpi_comm_world
```

Comment:
This is the only change in EnKF code. I am not sure this is needed. Please check.

Reply:
Thanks for your comments. We notice that "mpi_comm_world" is loaded twice respectively from mpisetup (line 46) and mpimod (line 47) modules. Our compilation on Orion failed due to this "mpi_comm_world" issue. So we removed it from line 46. Is this a code bug or does someone intend to load it twice in this subroutine?

Reply:
Thanks. Now I see this a a change needed for portability.